src/schema/v2/Feature/FeatureMeta.ts

```diff
@@ -0,0 +1,35 @@
+import { GraphQLNonNull, GraphQLObjectType, GraphQLString } from "graphql"
+import { Gravity } from "types/runtime"
+import { ResolverContext } from "types/graphql"
+import { markdownToText } from "lib/helpers"
+
+const DEFAULT_META_DESCRIPTION =
+  "Artsy is the world’s largest online art marketplace. Browse over 1 million artworks by iconic and emerging artists, presented by 4000+ galleries and top auction houses in over 100 countries."
+
+export const FeatureMetaType = new GraphQLObjectType<
+  Gravity.Feature,
+  ResolverContext
+>({
+  name: "FeatureMeta",
+  description: "Meta-tag related fields for Features",
+  fields: () => ({
+    name: {
+      type: new GraphQLNonNull(GraphQLString),
+      resolve: ({ name }) => `${name} | Artsy`,
+    },
+    description: {
+      type: new GraphQLNonNull(GraphQLString),
+      resolve: ({ description }) => {
+        if (!description) return DEFAULT_META_DESCRIPTION
```

Comment:
Any tests?

Reply:
🤷

Reply:
Really though: I can add some — but Features are the only piece actually guarded by a runtype and the corresponding typing information is threaded through here. So you can have a lot of confidence in the typing.

Reply:
https://github.com/artsy/metaphysics/blob/9e77f3b8ee1edb9fdbd2c8ec486b02b1bdf341b1/src/lib/loaders/loaders_without_authentication/gravity.ts#L32

Reply:
There's probably a larger debate here around swapping a long-term culture of tests for types that we should raise somewhere, though the angle makes sense.

Reply:
I suppose my thinking is that tests on GraphQL fields that don't have logic complex enough to extract is a lot like snapshot tests of React components. The value proposition doesn't quite add up to me: keeping a fixture in sync with the type, etc for what amounts to testing a single conditional when invalid data can't even make it to this resolver at runtime.

Though the expectation that any meta description falls back to a logical default is a totally reasonable thing to have in a spec — I'm going to be working on these types a bit coming up so I'll be sure to add some basic coverage.

Reply:
ty 👍 It's not that one tests every bit of logic, but that behavior is tested. A single conditional is easy to reason about, but from a public API perspective that's irrelevant. If we have x number of clients expecting a default and someone comes by and changes that, that breaks consumers.